android/app/src/main/kotlin/io/devopen/subzilla/widget/SubsGlanceWidget.kt

```
@@ -0,0 +1,340 @@
+            style = TextStyle(fontSize = 12.sp, color = GlanceTheme.colors.onBackground),
+            modifier = GlanceModifier.defaultWeight(),
+            maxLines = 1
+        )
+        Spacer(modifier = GlanceModifier.width(8.dp))
+        Text(
+            daysLabel,
+            style = TextStyle(fontSize = 10.sp, color = urgencyColorProvider(days))
+        )
+        Spacer(modifier = GlanceModifier.width(10.dp))
+        Text(
+            "$currency${String.format("%.2f", sub.amount)}",
```

detekt (1.23.8)

[warning] String.format("%.2f", sub.amount) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)